### ISSUE TITLE
Spamming the next key will mess up character position updates.

### ISSUE THREAD

**Issue by coppolaemilio** [coppolaemilio/dialogic#1208]
As you can see at the start of the video, it all goes well when you skip the text at a reasonable speed, but after I start spamming the next key the position gets messed up.

https://user-images.githubusercontent.com/2206700/188137122-f187da27-5741-451b-8ee8-fcf8c825f865.mp4

To fix this, there should be a different internal variable (like `target_postion`), so when updating the position we don't interpolate the variable and we are sure that the final position will be the expected one.

So basically, use `target_position` internally and check if the portrait position is not equal to `target_position` move it there.

**Comment by Jowan-Spooner** [comment]
I don't actually think this is an error, right? It's just doing it very quickly. Keep in mind that the Set position event is used weirdly in the example timeline, meaning it will shift the position each time + 100 on the x axis. If you wouldn't want this you would have to add a Set position event that's set to reset.

**Comment by KatieZeldaKat** [comment]
I agree with @Jowan-Spooner here, it seems as though position 1 is changing by 100 pixels each time the method is called. It even seems as though you can see the change happen when you aren't spamming the button. The character's first position when saying "You are right" is to the very far left of the screen, and when circling back to the same dialog, the character is now a little bit from the screen's edge.
